### PR TITLE
test: explain tests for queries whose data come from both parquet and ingester

### PR DESCRIPTION
**Pull request by NGA-TRAN**
- [x] I've read the contributing section of the project [README](https://github.com/influxdata/influxdb_iox/blob/main/README.md).
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed).

### CODE REVIEW

File: query_tests/cases/in/new_sql_system_tables.expected [hunk 1]
```diff
@@ -1,4 +1,4 @@
--- Test Setup: NewTwoMeasurementsManyFieldsTwoChunks
+-- Test Setup: TwoMeasurementsManyFieldsTwoChunks
```
Comment (NGA-TRAN) [inline query_tests/cases/in/new_sql_system_tables.expected:1]: Remove `New` as we no longer needs it

File: query_tests/src/scenarios/library.rs [hunk 2]
```diff
@@ -391,9 +391,9 @@ impl DbSetup for TwoMeasurementsManyFieldsOneChunk {
 /// This has two chunks for queries that check the state of the system
 ///
 /// This scenario is NG-specific and can be used for `EXPLAIN` plans and system tables.
-pub struct NewTwoMeasurementsManyFieldsTwoChunks {}
+pub struct TwoMeasurementsManyFieldsTwoChunks {}
```
Comment (alamb) [inline query_tests/src/scenarios/library.rs:394]: 👍

File: query_tests/cases/in/two_chunks.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+-- IOX_SETUP: TwoMeasurementsManyFieldsTwoChunks
+
+-- validate we have access to information schema for listing system tables
+-- IOX_COMPARE: sorted
+SELECT * from h2o;
+-- Plan will look like:
```
Comment (alamb) [inline query_tests/cases/in/two_chunks.sql:6]: I verified that the plans match this textual description 👍

File: query_tests/cases/in/two_chunks.expected [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
++---------------+---------------------------------------------------------------------------------------------------------------------+
+-- SQL: select temp, other_temp, time from h2o;
++------+------------+--------------------------------+
+| temp | other_temp | time                           |
++------+------------+--------------------------------+
+|      | 72.4       | 1970-01-01T00:00:00.000000150Z |
+| 70.4 |            | 1970-01-01T00:00:00.000000050Z |
+|      | 70.4       | 1970-01-01T00:00:00.000000250Z |
++------+------------+--------------------------------+
+-- SQL: EXPLAIN select temp, other_temp, time from h2o;
```
Comment (alamb) [inline query_tests/cases/in/two_chunks.expected:34]: what is the rationale for both `select *` and `select temp, other_temp, time...` from the same table? Other than the projection the plans look the same, right?

File: query_tests/cases/in/several_chunks.sql [hunk 5]
```diff
@@ -0,0 +1,14 @@
+-- IOX_SETUP: ManyFieldsSeveralChunks
+
+-- validate we have access to information schema for listing system tables
+-- IOX_COMPARE: sorted
+SELECT * from h2o;
+-- Plan will look like:
+--  . Two chunks (one parquet and one from ingester) neither overlap nor contain duplicate
+--     --> scan in one scan node
```
Comment (alamb) [inline query_tests/cases/in/several_chunks.sql:8]:
```suggestion
--  . One chunk from ingester, neither overlap nor contain duplicate
```

I didn't quite follow how there were two chunks in this scenario

**Review by alamb — APPROVED** [review]
Thanks @NGA-TRAN  this is looking very good

File: query_tests/cases/in/several_chunks.sql (hunk 5 above)
Comment (alamb) [reply query_tests/cases/in/several_chunks.sql:8]: Ah, I see 

> |               |     IOxReadFilterNode: table_name=h2o, chunks=2 predicate=Predicate                   |

There are two chunks there

**Pull request merged**